README.md

```diff
@@ -8,11 +8,12 @@ This demo starts an OpenShift cluster configured to authenticate with Keycloak.
 
 There is some configuration of the demo available in the `config` file.
 
-## Build OpenShift
+## Use OpenShift Client Tools v.3.10+
 
-OpenShift 3.9 does not contain the features required to integrate with Keycloak. Until 3.10 is released you have to build OpenShift from source. For details on how to do that check out the [OpenShift Origin Contributors guide](https://github.com/openshift/origin/blob/master/CONTRIBUTING.adoc#develop-locally-on-your-host).
+OpenShift 3.9 does not contain the features required to integrate with Keycloak. In order to run this demo, you need
+to use the latest version of `oc` client (at the moment of writing this doc, it's v3.10.0-rc.0). The other alternative is
+building `oc` locally (but this is beyond of the scopr of this doc).
```

Comment:
scopr -> scope

Reply:
Done

Reply:
where? :-)

Reply:
hahaha Please wait a couple of minutes.. PR is going to be updated :D 

It's my old habit of replying to comments while still editing it locally. You're just too quick!